backend-app/package.json

```diff
@@ -39,6 +39,10 @@
     "yamljs": "^0.3.0"
   },
   "devDependencies": {
-    "nodemon": "^2.0.22"
+    "eslint": "^8.45.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "nodemon": "^2.0.22",
+    "prettier": "^3.0.0"
   }
```

Comment:
can you quickly  use specified version instead of ^ because it may cause a different in the future